### PR TITLE
Enable depth peeling in default config

### PR DESCRIPTION
**Pull request by Meakk**
Fix https://github.com/f3d-app/f3d/issues/1046

Additionally, a small margin is added to the size to avoid z-fighting if large translucent triangles are exactly aligned with the grid bounds

### CODE REVIEW

**Comment by mwestphal** [comment]
many more baselines to update

File: library/VTKExtensions/Rendering/vtkF3DRenderer.cxx [hunk 1]
```diff
@@ -498,7 +498,10 @@ void vtkF3DRenderer::ConfigureGridUsingCurrentActors()
       {
         for (int i = 0; i < 3; i++)
         {
-          double size = bounds[2 * i + 1] - bounds[2 * i];
+          // a small margin is added to the size to avoid z-fighting if large translucent
+          // triangles are exactly aligned with the grid bounds
+          constexpr double margin = 1.0001;
+          double size = margin * (bounds[2 * i + 1] - bounds[2 * i]);
```
Comment (mwestphal) [inline library/VTKExtensions/Rendering/vtkF3DRenderer.cxx:504]: what does this do ? please mention it in the PR description

File: testing/baselines/TestDefaultConfigTranslucent.png
Comment (mwestphal) [inline testing/baselines/TestDefaultConfigTranslucent.png:1]: I dont see much transparency in this image though

**Review by mwestphal — CHANGES_REQUESTED** [review]
discussion

File: library/VTKExtensions/Rendering/vtkF3DRenderer.cxx (hunk 1 above)
Comment (Meakk) [reply library/VTKExtensions/Rendering/vtkF3DRenderer.cxx:504]: There's a comment in the code

File: testing/baselines/TestDefaultConfigTranslucent.png
Comment (Meakk) [reply testing/baselines/TestDefaultConfigTranslucent.png:1]: It's done in purpose. Most of the issues reported by the users are just opaque materials with a BLEND tag in the gltf file.  
If you disable depth peeling on the new test case file, you'll see the problem

File: library/VTKExtensions/Rendering/vtkF3DRenderer.cxx (hunk 1 above)
Comment (mwestphal) [reply library/VTKExtensions/Rendering/vtkF3DRenderer.cxx:504]: > please mention it in the PR description

and make the life of the person creating the release easier.

Alternatively update the changelog, but that ship has sailed ^^

**Review by mwestphal — APPROVED** [review]

**Pull request merged**